src/components/organisms/topbar/project/projectName.tsx

```diff
@@ -12,15 +12,18 @@ import { useProjectStore, useToastStore } from "@store";
 import { ErrorMessage } from "@components/atoms";
 import { CopyButton } from "@components/molecules";
 
+import { EditIcon } from "@assets/image/icons";
+
 export const ProjectTopbarName = () => {
 	const { projectId } = useParams();
 	const { getProject, renameProject } = useProjectStore();
 	const [isNameValid, setIsNameValid] = useState(true);
 	const [project, setProject] = useState<Project>();
+	const [isEditing, setIsEditing] = useState(false);
 	const { t } = useTranslation(["projects", "buttons"]);
 	const { t: tErrors } = useTranslation("errors");
 	const inputClass = cn(
-		"min-w-3 rounded bg-transparent p-0 text-xl font-bold leading-6 leading-tight outline outline-0",
+		"min-w-3 rounded bg-transparent p-0 text-xl font-bold leading-6 leading-tight outline outline-0 max-w-120 transition focus:bg-gray-1200",
```

Comment:
Please make it `max-width` 240 on normal screens
and `max-width` 160 on screen smaller than 1600px

Reply:
changed